lib/screens/v2/candidate_generator/widgets/full_line_map.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Screens.V2.CandidateGenerator.Widgets.FullLineMap do
+  @moduledoc false
+
+  alias Screens.Config.Screen
+  alias Screens.Config.V2.{PreFare, FullLineMap}
+  alias Screens.V2.WidgetInstance.FullLineMap, as: FullLineMapWidget
+  alias Screens.Util.Assets
+
+  def line_map_instances(%Screen{app_params: %PreFare{full_line_map: full_line_maps}} = config) do
```

Comment:
Sorry, one last nitpicky thing. Do you mind changing this function and functions with similar names to match the widget name? Normally it wouldn't be a big deal but we don't want to confuse this with the e-ink line map.

Reply:
Oh definitely, thanks for that catch!